go-fuzz-build/main.go

```diff
@@ -737,18 +751,41 @@ func (c *Context) instrumentPackages(blocks *[]CoverBlock, sonar *[]CoverBlock)
 }
 
 func (c *Context) copyDir(dir, newDir string) {
-	c.mkdirAll(newDir)
 	files, err := ioutil.ReadDir(dir)
 	if err != nil {
 		c.failf("failed to scan dir '%v': %v", dir, err)
 	}
+	c.mkdirAll(newDir)
 	for _, f := range files {
+		src := filepath.Join(dir, f.Name())
```

Comment:
Would you mind doing cleanup in a separate PR? This is a nice change; if you send a cleanup PR   first we can merge that quickly and easily, and make it clearer what the actual changes are.

Reply:
OK, see the https://github.com/dvyukov/go-fuzz/pull/312